labs-ios-starter/Surveys/View Controllers/TopicNameViewController.swift

```diff
@@ -39,10 +99,32 @@ class TopicNameViewController: UIViewController {
                 return
             }
             questionsVC.topicName = topicName
+            let index = contextPicker.selectedRow(inComponent: 0)
+            let contextId = Int(contexts?[index].id ?? 0)
+
+            questionsVC.contextID = contextId
         }
     }
 }
 
+extension TopicNameViewController: UIPickerViewDelegate, UIPickerViewDataSource {
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        1
```

Comment:
```suggestion
        return 1
```
Can format for consistency. One-liners have been using return keyword so far

Reply:
I guess I won't ask us to strip them all out, but that's the way I prefer the one-liners lol. No worries